portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -40,7 +42,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   private String getParameter(HttpServletRequest request, String name, String defaultValue) {
     String value = request.getParameter(name);
     
-    if (value.equals("")) {
+    if (value.equals("") || value == null) {
```

Comment:
I think you will want to perform the null check before you try to compare it with the empty string.

Reply:
Good point, I fixed this. Thanks!